google-cloud-container/acceptance/google/cloud/container/v1/cluster_manager_smoke_test.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# AUTO GENERATED BY GAPIC
+
+require "minitest/autorun"
+require "minitest/spec"
+
+require "google/cloud/container"
+
+describe "ClusterManagerSmokeTest" do
+  it "runs one smoke test with list_clusters" do
+    unless ENV["CONTAINER_TEST_PROJECT"]
+      fail "Usage: CONTAINER_TEST_PROJECT=<project_id> ruby #{$0}"
```

Comment:
Why is it necessary to require that `CONTAINER_TEST_PROJECT` be set?

In every package (including this one), the Rakefile `acceptance` task also accepts the more general `GCLOUD_TEST_PROJECT`:

```ruby
project ||= ENV["GCLOUD_TEST_PROJECT"] || ENV["CONTAINER_TEST_PROJECT"]
```

Reply:
In this particular case it's not necessary, but part of the generator that creates the Rakefile is not currently smart enough to know that the smoke test doesn't need to read this variable.

I think we can probably just remove this check from the generator and let smoke tests that do require the env var to fail at runtime.